frontend/src/components/AppMap.vue

```diff
@@ -143,7 +149,7 @@ type Props = {
   data?: Data;
   overlays?: Overlays;
   /** map of feature id to value */
-  values?: { [key: number]: number };
+  values?: { [key: number]: { v: number, aac?: number } };
```

Comment:
This can be changed to `[key: string]` as well. Idk why a TS error isn't being thrown for passing `[key: string]` to `[key: number]`.